modules/utxo-lib/src/address.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * @prettier
+ */
+import * as bitcoinjs from 'bitcoinjs-lib';
+
+import * as zcashAddress from '../src/bitgo/zcash/address';
```

Comment:
`zcash` and it's two-byte version prefix are the only reason why this file is needed